lib/fastlane/plugin/icon_versioning/helper/version_icon_helper.rb

```diff
@@ -81,8 +82,8 @@ def run()
         File.open(cache_file_path, 'w') { |file| file.write(cache.to_yaml) }
       end
 
-      def self.get_versioned_path(path)
-        return path.gsub(/([^.]+)(\.appiconset)/, '\1-Versioned\2')
+      def self.get_versioned_path(path, suffix)
```

Comment:
This is called from `icon_versioning_action_spec.rb` and `version_icon_action_spec.rb` too